txn.go

```diff
@@ -514,9 +514,11 @@ func (txn *Txn) commitUpdates(chunk commit.Chunk) (updated bool) {
 
 			// Range through all of the pending updates and apply them to the column
 			// and its associated computed columns.
+			txn.owner.lock.Lock()
```

Comment:
Wouldn't the chunk be already locked? I'm afraid that we'd limit concurrency significantly as only a single writer would be allowed to apply the transaction concurrently in this case. Now, that might not be too bad but I wonder if there's some better way instead of locking the entire Apply?

Reply:
Unfortunately, I'm not able to work around locking the fill map completely - but I think I've come up with a quicker method that still (seems) to fix the race.